docs/data-sources/cce_nodes.md

```diff
@@ -0,0 +1,92 @@
+---
+subcategory: "Cloud Container Engine (CCE)"
+---
+
+# huaweicloud_cce_nodes
+
+Use this data source to get a list of CCE nodes.
+
+## Example Usage
+
+```hcl
+variable "cluster_id" {}
+variable "node_name" {}
+
+data "huaweicloud_cce_nodes" "node" {
+  cluster_id = var.cluster_id
+  name       = var.node_name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to obtain the cce nodes. If omitted, the provider-level
+  region will be used.
+
+* `cluster_id` - (Required, String) Specifies the id of cce cluster.
```

Comment:
ID is an abbreviation of proper nouns, suggest to use the `ID` to indicate.

Reply:
fixed